opencxl/cxl/device/downstream_port_device.py

```diff
@@ -47,63 +49,74 @@
 )
 
 
+class SleepLoop(RunnableComponent):
+    def __init__(self):
+        super().__init__()
+        self._task = None
+        self._running = False
+        self._loopcond = Condition()
+
+    async def _process(self):
+        await self._loopcond.acquire()
+        try:
+            await self._change_status_to_running()
+            if self._running:
+                await self._loopcond.wait()
+        finally:
+            self._loopcond.release()
+
+    async def _run(self):
+        self._running = True
+        self._task = create_task(self._process())
+        await self.wait_for_ready()
+        await self._task
+
+    async def _stop(self):
+        self._running = False
+        await self._loopcond.acquire()
+        self._loopcond.notify()
+        self._loopcond.release()
+        await self._task
+
+
 class DownstreamPortDevice(CxlPortDevice):
     def __init__(
         self,
         transport_connection: CxlConnection,
         port_index: int = 0,
-        ld_count: int = 1,
+        max_ld: int = 1,
```

Comment:
Could you explain the need for this variable?

Reply:
Nothing fancy, just a sanity bound check in https://github.com/opencxl/opencxl-core/pull/130/files/b4260b3d9afa1b1413b4795860f39e696660485d..96bcb3e82acd7f0adda18b63da70096596734d7b#diff-01bc0503515a53f29c1aa7e493ec3902dac6603491e78da934945b4486da3713R183

We can get rid of it if you want.

Reply:
Removed.